src/clr-angular/forms/combobox/_combobox.clarity.scss

```diff
@@ -1,54 +1,128 @@
-// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
 @import '../../popover/dropdown/menu-mixins.clarity';
 @import './properties.combobox';
 
 @include exports('combobox.clarity') {
-  .clr-combobox {
-    display: inline-block;
+  .clr-aria-only {
+    display: none;
   }
 
-  .clr-combobox-input-container {
-    display: flex;
-    align-items: center;
-  }
+  .clr-combobox-container {
+    position: relative;
 
-  .clr-combobox-input {
     //Dimensions
     padding: 0 $clr-ng-select-trigger-right-padding 0 $clr-ng-select-trigger-left-padding;
     min-height: $clr-ng-select-min-height;
-    width: $clr-ng-select-min-width;
+    min-width: $clr-ng-select-min-width;
     @include css-var(font-size, clr-ng-select-font-size, $clr-ng-select-font-size, $clr-use-custom-properties);
 
+    &.multi {
+      min-width: $clr-ng-multiselect-min-width;
+      padding-bottom: $clr_baselineRem_0_125;
+    }
+
     //Mixins
-    @include form-fields-appearance();
-    @include input-border-bottom-animation();
+    @include form-fields-appearance($clr-forms-border-color);
+
+    // override the above mixin for combobox-specific styles
+    background: hsl(201, 10%, 96%); // #F4F5F6
+    border-bottom: $clr-global-borderwidth solid $clr-forms-border-color;
+    //border-radius: $clr-default-borderradius;
+
+    //@include input-border-bottom-animation($clr-forms-focused-color);
+
+    // ideally I want to just use &:focus-within, but cross-browser support is limited
+    // for now, we programmatically add/remove .clr-focus class to achieve this
+    &:focus-within,
+    &.clr-focus {
+      border-bottom: $clr_baselineRem_2px solid $clr-forms-focused-color;
+    }
 
     //Others
-    outline: none;
+    .clr-combobox-input {
+      background: none;
+      border-bottom: none;
+
+      &:focus,
+      &.clr-focus {
+        background: none !important;
```

Comment:
Why so `important` here and also, why not `inherit`?

Reply:
I absolutely have no idea why it is there. Not inside it's parent like this:
```
  //Others
    .clr-combobox-input {
      background: none;
      border-bottom: none;

    //   &:focus,
    //   &.clr-focus {
    //     background: none !important;
    //     border-bottom: none !important;
    //   }
    }
```
Removing. It does not seem to do anything meaningful as I test it.